lib/config.js

```diff
@@ -7,30 +7,31 @@ var log = require('npmlog')
 var mkdirp = require('mkdirp')
 
 module.exports = function (options) {
-  var projectPath = options.path || process.cwd()
```

Comment:
I couldn’t find any place where `options.path` is used, I think it was part of our old Hoodie’s test setup.